pyproject.toml

```diff
@@ -20,15 +20,12 @@ click = "^8.1.3"
 fastapi = { version = "^0.89.1", extras = ["all"] }
 gql = { version = "^3.4.0", extras = ["all"] }
 python = "^3.8"
-pillow = "^9.0.1"
 prompt-toolkit = "^3.0.31"
 PyYAML = "^6.0"
```

Comment:
do we use this? I import straight up `yaml`

Reply:
Not sure. Removing it doesn't affect poetry.lock at all. Went ahead and removed this one as well.